app/networkapi/people/models.py

```diff
@@ -92,6 +93,18 @@ class Person(models.Model):
         help_text='Do you want to feature this person?',
         default=False,
     )
+    publish_after = models.DateTimeField(
+        help_text='Publish this person after this date and time (UTC)',
```

Comment:
Let's reword this to `Make this person's profile visible only after this date and time (UTC)` and for expires `Hide this person's profile after this date and time (UTC)`